Remotion/ObjectBinding/Web/Res/Themes/NovaViso/HTML/BocList.css

```diff
@@ -1,594 +1,333 @@
-div.bocList.hasDimensions div.bocListTableBlock.hasMenuBlock
-{
-  right: 10em; /* width of BocList menu block + some space */
-}
-
-/* Style applied to the div surrounding the BocList's table. */
-div.bocListTableContainer
+/*region bocListTableScrollContainer*/
+div.bocList div.bocListTableBlock div.bocListTableContainer div.bocListTableScrollContainer
 {
+  background-color: var(--color-standard-background);
+  scrollbar-width: var(--remotion-themed-scrollbar-width);
+  scrollbar-color: var(--color-standard-scrollbar-track);
   display: block;
-  width: auto;
-  height: auto;
-  background-color: #F0F0F0;
+  overflow: auto;
 }
 
-div.bocList.hasDimensions div.bocListTableBlock div.bocListTableContainer
+div.bocList.hasDimensions div.bocListTableBlock div.bocListTableContainer div.bocListTableScrollContainer
 {
   position: absolute;
-  top: 0;
   left: 0;
   bottom: 0;
   right: 0;
-  overflow: hidden;
+  border-top: 1px solid var(--color-standard-border);
 }
 
-div.bocList.hasDimensions div.bocListTableBlock span:focus + div.bocListTableContainer
+div.bocList div.bocListTableBlock div.bocListTableContainer div.bocListTableScrollContainer::-webkit-scrollbar-thumb
 {
-  outline: dotted black 1px;
-  outline: auto 5px -webkit-focus-ring-color;
+  background: var(--color-standard-scrollbar-track);
 }
 
-/*  Style applied to the div surrounding the BocList's table when the BocList has a defined width or height and a navigation block.  */
-div.bocList.hasDimensions div.bocListTableBlock.hasNavigator div.bocListTableContainer
+div.bocList div.bocListTableBlock div.bocListTableContainer div.bocListTableScrollContainer::-webkit-scrollbar
 {
-  bottom: 1.85em;
+  width: var(--remotion-themed-scrollbar-width);
 }
+/*endregion*/
 
-/* Style applied to the div allowing the BocList's table to scroll. */
-div.bocListTableScrollContainer
+/*region cells*/
+div.bocList table.bocListTable th.bocListTitleCell,
+div.bocList table.bocListTable td.bocListDataCell
 {
-  display: block;
-  width: auto;
-  height: auto;
-  overflow-x: auto;
-  overflow-y: hidden;
-  background-color: White;
-}
-
-div.bocList.hasDimensions div.bocListTableScrollContainer
-{
-  position: absolute;
-  top: 1.9em;
-  left: 0;
-  bottom: 0;
-  right: 0;
-  overflow: auto;
-  border-top: 1px solid #AAAAAA;
+  border-top: none;
+  border-right: none;
+  border-left: none;
+  padding: var(--remotion-themed-spacing-half) var(--remotion-themed-spacing);
 }
 
-div.bocList.hasDimensions div.bocListTableScrollContainer table.bocListTable
+div.bocList table.bocListTable th.bocListTitleCell
 {
-  margin-top: -1.9em;
-  position: relative;
+  font-weight: normal;
+  text-align: left;
+  background-color: var(--boclist-tablehead-background-color);
+  white-space: nowrap;
 }
 
-
-/* Style applied to the fake table head used for scrolling */
-div.bocListFakeTableHead
+div.bocList table.bocListTable th.bocListTitleCell > input[type=checkbox]
```

Comment:
need to cover checkboxes in the datarows, too. otherwise, they're not aligned.
also, radiobuttons in the datarows.